packages/skipper/src/HttpKernel/PathNormalizerKernel.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\Skipper\HttpKernel;
+
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+use Symplify\Skipper\Bundle\PathNormalizerBundle;
+use Symplify\SymplifyKernel\Bundle\SymplifyKernelBundle;
+use Symplify\SymplifyKernel\HttpKernel\AbstractSymplifyKernel;
+
+final class PathNormalizerKernel extends AbstractSymplifyKernel
```

Comment:
```suggestion
final class SkipperKernel extends AbstractSymplifyKernel
```

This is the only extra class needed for tests